### PR TITLE
UCT/IB/UD: Add configurable slow timer tick

### PR DESCRIPTION
**Pull request by dmitrygladkov**
## What

Add configuration parameter for IB/UD slow timer tick

## Why ?

Address TODO in the code + ports changes from #3603

## How ?

Read the user passed value and convert to `ucs_time_t`

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10651/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10652/ for details (Mellanox internal link).

File: src/uct/ib/ud/base/ud_iface.c [hunk 1]
```diff
@@ -553,6 +559,8 @@ ucs_config_field_t uct_ud_iface_config_table[] = {
 
     {"TIMEOUT", "5.0m", "Transport timeout",
      ucs_offsetof(uct_ud_iface_config_t, peer_timeout), UCS_CONFIG_TYPE_TIME},
+    {"SLOW_TIMER_TICK", "100ms", "Initial time granularity for resending trigger",
```
Comment (yosefe) [inline src/uct/ib/ud/base/ud_iface.c]: "Initial timeout for retransmissions"

Comment (dmitrygladkov) [reply src/uct/ib/ud/base/ud_iface.c]: done

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10658/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 24-Jun-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/3e9a0bc99c9343ce062464604d258b842543e9d8/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/3abf7504a3d99001a2e9bf3a5bb63273ad5cf165/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/483aae00939dc8e4ded97a8f4da6197bb50011a9/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/d7ba4f06274cb6da6d9d34967050c980658f9d46/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/fd3bc12ac72b237c657f20e6d8a557ed23ccd1df/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/97a396786d94fedd83e253f769e571a0388987d1/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/02d94b1fc5c9e43f892e404b2a1940d689e2e16f/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/af6c754b7a470a57ff1fc2c3f1e25d5c6fc81739/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/b3ada88a353878a2e6def51dfed91c137645ee8d/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/3dfdb772c72fd884bdfaa83a5c797d3fa8eadfc5/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/103afb3a7f81203ad71b0c18488dd83841f6541c/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/764164ff9b4ec90920078e1599dda516ee9798af/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/d268bed488038cd91b44ae916bcb335206392c0c/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W0](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/cf8645e7f07c9e783e55e4ec34d20e51116b278d/hpc-test-node2_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W1](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/1946b29a73752fd4e1b6ff70f6f606411f9e342a/hpc-test-node2_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W2](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/13e12cfc88f2562bc09aa55029c6a9e4b4a32a1d/hpc-test-node2_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W3](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/5436895232643baaee8283c431498992879de012/hpc-test-node2_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W0](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/6635427a916f758ca7563a7fd864f61df214e9b9/hpc-test-node3_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W1](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/099c0aa6065cae3c1072e2b4f16d7706f4c631f3/hpc-test-node3_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W2](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/0d9af3906a21539b70bd884df52e3f0177cebb3b/hpc-test-node3_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W3](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/7e6514fe064083f78ce29e494d1c7cca9ff10cd9/hpc-test-node3_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node_W0](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/4a3a68ae347fef024dabb12ed5ab56ecae39dd1e/hpc-test-node_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node_W1](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/4e86fea761f01fd75f0f3c3ef114998e2c1bc5ca/hpc-test-node_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node_W2](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/b09e522a8303ce707858ec91ab1b427867ad99ff/hpc-test-node_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node_W3](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/be0c4806bb8415e0541822b72215bdb4dee76990/hpc-test-node_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/104b672176ea5485c688ece6b626f55e26aeed2c/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/cc7bfaf5117d9440cece95bac32218cb1f0bc9ad/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/1359b8270c66573016dca3e1972b33798aeedf09/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/c1d9a4e07c29d8b92c67de4421a47729bdfbb7e3/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/2e40383d77ac60268278429c69ce75b65d11c529/vulcan-jenkins_W0) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/5683e236ebdb0dfd4f7fa339bd9ea130fac8a0f2/vulcan-jenkins_W1) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/5a616db8112659a69eac4003ba58876a05acd5c9/vulcan-jenkins_W2) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/43d46f1833240de020c087f2da297ddc/raw/d7d795aebdc639c74f96415213be79553a689c24/vulcan-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by dmitrygladkov** [comment]
@yosefe ok to merge?

**Pull request merged**